docs/conf.py

```diff
@@ -91,14 +91,27 @@
     'source/*.FA_2_15.*',
     'source/*.FA_2_16.*',
 #    'source/*.FA_2_17.*',
-#    'source/*.FA_2_18.*',
-#    'source/*.FA_2_19.*',
-#    'source/*.FA_2_20.*',
-#    'source/*.FA_2_21.*',
+    'source/*.FA_2_18.*',
+    'source/*.FA_2_19.*',
+    'source/*.FA_2_20.*',
+    'source/*.FA_2_21.*',
+    'source/*.FA_2_22.*',
+    'source/*.FA_2_23.*',
+    'source/*.FA_2_24.*',
+    'source/*.FA_2_25.*',
```

Comment:
Hmm are we adding more versions instead of trimming it down?

Reply:
We are adding them to exclude list. Unfortunately sphinx does not provide ability to specify what we want to include into the docs, it only allows to exclude some patterns.